swap/src/protocol/alice/behaviour.rs

```diff
@@ -90,6 +96,9 @@ where
         resume_only: bool,
         env_config: env::Config,
     ) -> Self {
+        let ping_config = PingConfig::new();
+        let ping = libp2p::ping::Ping::new(ping_config);
```

Comment:
Nit: You are being inconsistent in what you import directly (`PingConfig` & `PingEvent`) with what you fully qualify (`libp2p::ping::Ping`). Also, `Ping` actually implements `Default` so you could do all of this in one line directly where `Self` is initialized :)